libraries/AC_AutoTune/AC_AutoTune.h

```diff
@@ -145,14 +194,19 @@ class AC_AutoTune {
         RD_UP = 0,                // rate D is being tuned up
         RD_DOWN = 1,              // rate D is being tuned down
         RP_UP = 2,                // rate P is being tuned up
-        SP_DOWN = 3,              // angle P is being tuned down
-        SP_UP = 4                 // angle P is being tuned up
+        RP_DOWN = 3,              // rate P is being tuned down
+        RFF_UP = 4,               // rate FF is being tuned up
+        RFF_DOWN = 5,             // rate FF is being tuned down
+        SP_UP = 6,                // angle P is being tuned up
+        SP_DOWN = 7,               // angle P is being tuned down
```

Comment:
extra space here.  Also may be add a comment beside MAX_GAINS enum value

Reply:
got it. thanks!